searx/engines/docker_hub.py

```diff
@@ -0,0 +1,69 @@
+"""
+Docker Hub (IT)
+
+@website     https://hub.docker.com
+@provide-api yes (https://docs.docker.com/registry/spec/api/)
+
+@using-api   yes
+@results     JSON
+@stable      yes
+@parse       url, title, content, thumbnail, publishedDate
+"""
```

Comment:
Please use 'about' notation / like any other engine does, here is an example from the google engine:

https://github.com/searx/searx/blob/ab8739809c8c5f4c1f12e4b15fa8c61afe30ef9f/searx/engines/google.py#L22-L29

Reply:
The wikidata_id is Q100769064 : https://www.wikidata.org/wiki/Q100769064